Subsurface/Subsurface/SAppDelegate.m

```diff
@@ -151,4 +163,24 @@ - (void)registerDefaultsFromSettingsBundle {
     [userDefaults synchronize];
 }
 
+- (void)scheduleNotification {
+	self.batteryNotification = [[UILocalNotification alloc] init];
+	self.batteryNotification.fireDate = [NSDate dateWithTimeIntervalSinceNow: 10];
+	self.batteryNotification.alertBody = @"Debugging- 5";
```

Comment:
@"Debugging- 5" strings seems too technical. Maybe some better string that describes what for it is?